identity-service/scripts/dev-server.sh

```diff
@@ -6,12 +6,25 @@ link_libs=true
 
 if [ "$link_libs" = true ]
 then
+    # Create anchor data link
+    cd ../solana-programs/anchor/audius-data
+    npm run dev &
+    npm link
+    cd ../../
+
+    # Link anchor data to libs
     cd ../audius-libs
+    npm link @audius/anchor-audius-data
+
+    # Create libs link
     npm run dev &
     npm link
+
+    # Link libs to app
```

Comment:
I think we agreed to link and build outside of the container and must mount the files